dependencies.go

```diff
@@ -11,6 +11,11 @@ import (
 
 //nolint:gochecknoglobals
 var (
+	// matches any sequence of characters enclosed by '/*' and '*/' including new lines and '*' not followed by '/'
+	reMultiLineComment = regexp.MustCompile(`\/\*(?:[^\*]|\*[^\/]|\n)*\*\/`)
```

Comment:
I think this won't match the documentation comments (/**)

Although it's unlikely that this will be a problem.

Reply:
It does indeed match them. I tested, and the corner case is if this comment is closed using `**/` instead of `*/`. I fixed it
```suggestion
	reMultiLineComment = regexp.MustCompile(`\/\*(?:[^\*]|\*[^\/]|\n)*(\*)+\/`)
```